pandas_datareader/av/__init__.py

```diff
@@ -78,7 +78,7 @@ def _read_lines(self, out):
                     ".".format(self.symbols)
                 )
             else:
-                raise RemoteDataError()
+                raise RemoteDataError(out)
```

Comment:
Rather than just dump out, it would be better to explain that this is the content retrieved from the website and then show out. Basically a formatted string is needed around out.

Reply:
Good call, fixed!